language/stdlib/modules/LibraTimestamp.move

```diff
@@ -48,6 +48,123 @@ module LibraTimestamp {
     public fun is_genesis(): bool acquires CurrentTimeMicroseconds {
         !::exists<CurrentTimeMicroseconds>(0xA550C18) || now_microseconds() == 0
     }
+
+    // **************** SPECIFICATIONS ****************
+
+    /*
```

Comment:
Why not make a documentation comment out of this? You can also use `/**` instead of `///`